.circleci/get_package_version

```diff
@@ -30,17 +30,18 @@ if echo "$CIRCLE_TAG" | grep -Eq '^v([0-9]+\.?)+$'; then
 fi
 
 # Find the most recent release tag on master.
-TAG=$(git describe --match='v*' --abbrev=0 origin/master)
+TAG=$(git describe --tags --match='v*' --abbrev=0 origin/master)
 
-# If we're building in CircleCI, the version is "x.y.z+b<n>" where <n> is the
+# If we're building in CircleCI, the version is "x.y.<b>" where <b> is the
```

Comment:
Why omit `z` here?

Reply:
Our `app/build.gradle` expects `versionNumber` to be in the form _<major>.<minor>.<patch>_. If we want the build number to appear in both the `versionString` and `versionCode`, it has to go somewhere, and I figured replacing the _<patch>_ number of the latest release was the most innocuous way to do that.

The alternative would be to continue to append the build number to the original `versionNumber` but then (a.) `build.gradle` would need to be updated to permit that and (b.) the `versionCode` wouldn't change between dev builds. (Maybe that's actually what we want?)

Reply:
First, just to be clear on terminology, the fields are named `versionName` and `versionCode`.

The only practical effect of the `versionCode` is to prevent unintended downgrades.  The Play Store will only perform automatic updates that increase the `versionCode`.  As far as I know, only the Play Store enforces this; the phone doesn't care whether the `vesionCode` increases or not. Because we don't plan to ever publish development APKs to the Play Store, in some sense it doesn't matter what we use for dev builds.

Any of these choices are fine:
  - Set `versionCode` to 0 for all dev builds
  - When `versionName` is `x.y.z`, set `versionCode` to 1000000x + 10000y + z for all dev builds
  - When `versionName` is `x.y.z` at patchlevel p, set `versionCode` to 1000000x + 10000y + 100z + p for all dev builds
  - When `versionName` is `x.y.z`, set `versionCode` to 1000000x + 10000y + p for all dev builds

The first is the simplest (it's what `build.gradle` currently does, and I think it will work fine—the net effect is that any production release will supersede (automatically install over) any development release.